### DISCUSSION TITLE
Contents restrictions

### DISCUSSION THREAD

**Discussion by Jay-H-C** [vercel/community#2252] [Help] [answered]
### Summary

Where can I find Vercels policy on hosted contents?

I'd like to start a web site for a shooting club I belong to, does this violated Vercel's policy?

thank you.

**Comment by amyegan** [comment, marked as answer]
Hi @Jay-H-C! You can find more information in our [Fair Use Policy](https://vercel.com/docs/concepts/limits/fair-use-policy#)
